torch/_subclasses/meta_utils.py

```diff
@@ -309,6 +344,12 @@ def describe_tensor(self, t: torch.Tensor, recurse: bool = True):
             attrs=attrs,
             ctx=ctx,
             type=type_v,
+            # NB: even if functorch is enabled, don't actually save the
+            # interpreter stack here unless we are actually functorch wrapped;
+            # it's irrelevant for non-functorch stuff
+            functorch_stack=functorch_stack,
```

Comment:
nit: `functorch_stack=maybe_functorch_stack`? I was confused as to where we avoid saving the interpreter stack if we're not functorch wrapped (e.g. in the MetaTensorDesc ctor or above this line, and it turned out to be above this line)